packages/core/src/textures/resources/VideoFrameResource.ts

```diff
@@ -0,0 +1,29 @@
+import { BaseImageResource } from './BaseImageResource';
+
+type VideoFrame = any;
```

Comment:
Maybe we use [@types/dom-webcodecs](https://www.npmjs.com/package/@types/dom-webcodecs) for the `VideoFrame` type?